src/components/ContactDetail.css

```
@@ -0,0 +1,198 @@
+  .detail-label {
+    display: block;
+    font-size: 0.75rem;
+    color: #94a3b8;
+    margin-bottom: 0.25rem;
+  }
+  
+  .detail-value {
+    display: block;
+    font-size: 1rem;
+    color: var(--card-foreground);
+    word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)